client/src/components/Search/SearchForm.js

```diff
@@ -0,0 +1,54 @@
+/* global document */
+import i18n from 'i18n';
+import React, { PropTypes } from 'react';
+
+import FormBuilderLoader from 'containers/FormBuilderLoader/FormBuilderLoader';
+import { Collapse } from 'reactstrap';
+
+import classnames from 'classnames';
+
+
+const identifier = 'Admin.SearchForm';
+const searchButtonClasses = classnames(
+  'btn',
+  'btn-primary',
+  'search-form__submit',
+  'btn--no-text',
+);
+const clearButtonClasses = classnames(
+  'btn',
+  'btn-secondary',
+  'search-form__clear',
+);
+
+const searchLabel = i18n._t('Admin.SEARCH', 'Search');
+const clearLabel = i18n._t('Admin.CLEAR', 'Clear');
+
+/**
+ * Display an advanced filter form.
+ */
+const SearchForm = ({ expanded, onSearch, onClear, formSchemaUrl, id }) => (
+  <Collapse id={id} className="search-form" isOpen={expanded}>
```

Comment:
This should have a nested `div` (ie. `search__filter__form`) with the padding applied to that rather than the Collapse element itself, this fixes the jumpy sliding animation

Reply:
Done ... so pretty to look at it slide now.

Reply:
Yea, it was hurting my eyes before 😵